### PR TITLE
NEW : Now we can edit amount on vat and salaries clone action

### PR DESCRIPTION
**Pull request by atm-gauthier**

### CODE REVIEW

File: htdocs/salaries/card.php [hunk 1]
```diff
@@ -380,6 +380,10 @@
 		$object->paye = 0;
 		$object->id = $object->ref = null;
 
+		if (GETPOST('amount', 'alphanohtml')) {
```
Comment (eldy) [inline htdocs/salaries/card.php:383]: Must be GETPOSTISSET instead of GETPOST so we can set amount to 0

File: htdocs/compta/tva/card.php [hunk 2]
```diff
@@ -333,6 +333,10 @@
 		$object->id = $object->ref = null;
 		$object->paye = 0;
 
+		if (GETPOST('amount', 'alphanohtml')) {
```
Comment (eldy) [inline htdocs/compta/tva/card.php:336]: Must be GETPOSTISSET instead of GETPOST so we can set amount to 0

Comment (atm-gauthier) [reply htdocs/compta/tva/card.php:336]: Hi @eldy, on vat and salaries amount is mandatory, why do you want to be able to set it to 0 ?
image

Comment (atm-gauthier) [reply htdocs/compta/tva/card.php:336]: I used the same test than social charges, but in this one there is a "$object->check();" before create, then if amount is null, clone action fails

Comment (atm-gauthier) [reply htdocs/compta/tva/card.php:336]: @eldy did you see my answer ?

**Pull request merged**